distribution/src/main/resources/config/elasticsearch.yml

```diff
@@ -69,7 +69,7 @@ ${path.logs}
 #
 # Prevent the "split brain" by configuring the majority of nodes (total number of master-eligible nodes / 2 + 1):
 #
-#discovery.zen.minimum_master_nodes: 3
+#discovery.zen.minimum_master_nodes: CHANGE ME TO NUMBER_OF_MASTER_ELIGIBLE_NODES / 2 + 1
```

Comment:
We already say this in the comment immediately above this line, and I don't think we should be shouting at the user. Can we be gentler? What about simply leaving this blank?

Reply:
Sure, the reason for capitals was actually only because I started with `NUMBER_OF_MASTER_ELIGIBLE_NODES` as if it was an environment variable and then realised that might give the wrong impression that it is an env variable. will change to blank